docs/developers/01-getting-started/01-react-app-quickstart.md

```diff
@@ -8,6 +8,10 @@ To start integrating NEAR in your React application, you can use the [Create Rea
 If you are looking for a more step by step approach, check out the [full tutorial](/developers/getting-started/react-tutorial).
 :::
 
+## Create a NEAR Account
+
+Checkout the official docs to get started creating a NEAR account [here](https://docs.near.org/docs/develop/basics/create-account).
```

Comment:
Redirecting them to the react-tutorial from here may be distracting. So, I copied over the content from react-tutorial here instead.